backend/tests_configs.ini

```diff
@@ -342,6 +343,36 @@ email.notification.smtp.password = just_a_password
 website.base_url = http://localhost:6543
 color.config_file_path = %(here)s/color-test.json
 user.reset_password.validity = 5
+email.notification.enabled_on_invitation = False
+
+[functional_test_with_mail_test_sync_with_auto_notif]
```

Comment:
we do not need to test here if notifications are sent, we need to check "when a user is invited, the boolean value is True instead of False")

Reply:
User can be invited but not connected a first time yet. that the goal of this test.